PAWI_HS18_Projektdokumentation.tex

```diff
@@ -991,6 +999,14 @@ \subsubsection{Massnahmen}
 	\hline
 	7 & Applikation wird nach CleanCode Standards entwickelt und durch Teammitglieder validiert\\
 	\hline
+	8 & Es werden mehrere Messungen zusammengefasst und der Median deren berechnet\\
+	\hline
+	9 & In einer Initialisierungsphase wird der Kompass kalibriert\\
+	\hline
+	10 & Dem Benutzer wird die Möglichkeit gegeben die Initialisierungsphase manuell neu zu starten\\
+	\hline
+	11 & Das Laden des Gebäudemodells geschieht in der Initialisierungsphase, welche nicht abgeschlossen ist solange das Gebäude nicht verfügbar ist\\
```

Comment:
Achtung gemeint mit zulangsam in der Darstellung ist, dass es bei bewegungen zu Stocken beginnt.
Die Korrekte lösung hierbei währe wohl "Das Gebäudemodell wird mit vereinfachten Geometrischen Formen neu Modeliert". Sonst sehr gut 👍 :)

Reply:
Habe ich so angepasst, danke für die Blumen 😄